docs/backend/global-utils.md

```diff
@@ -0,0 +1,253 @@
+---
+html_meta:
+  "description": "Global functions defined in CMFPlone and plone.app.layout."
+  "property=og:description": "Global functions defined in CMFPlone and plone.app.layout."
+  "property=og:title": "Global utils and helpers"
+  "keywords": "global variables,portal_state,context_state,plone_view,portal_url,helper methods"
```

Comment:
Please add spaces after commas for legibility. Also insert "Plone", "Classic UI", and "templates".

```suggestion
  "keywords": "global variables,portal_state,context_state,plone_view,portal_url,helper methods"
```